applications/vanilla/views/categories/flat_all.php

```diff
@@ -0,0 +1,99 @@
+<?php if (!defined('APPLICATION')) exit();
+if (!function_exists('GetOptions'))
+    include $this->fetchViewLocation('helper_functions', 'categories');
+
+echo '<h1 class="H HomepageTitle">'.$this->data('Title').'</h1>';
+if ($Description = $this->Description()) {
+    echo wrap($Description, 'div', array('class' => 'P PageDescription'));
+}
+$this->fireEvent('AfterPageTitle');
+
+$CatList = '';
+$DoHeadings = c('Vanilla.Categories.DoHeadings');
+$MaxDisplayDepth = c('Vanilla.Categories.MaxDisplayDepth') + $this->data('Category.Depth', 0);
+$ChildCategories = '';
+
+$CategoryTree = $this->data('CategoryTree');
+$Categories = CategoryModel::flattenTree($CategoryTree);
+$this->EventArguments['NumRows'] = $Categories;
+
+echo '<ul class="DataList CategoryList'.($DoHeadings ? ' CategoryListWithHeadings' : '').'">';
+foreach ($Categories as $CategoryRow) {
+    $Category = (object)$CategoryRow;
```

Comment:
I'm hoping you can try this with WriteListItem() so it's more of a simple view like the table.